eclipse-emfcloud/modelserver-glsp-integration
# #47 #48 Update to latest GLSP version due to API breaks and fix unsubscribing

#47 Update to latest GLSP version due to API breaks 
- Refactoring of OperationHandler type hierarchy API, see also eclipse-glsp/glsp-server#187
  - Stay with executeOperation method as we manage command stack via Model Server
  - Merge interfaces and abstract operation handler classes
- Update GLSP and Model Server dependencies

Part of #47 (Will update project-template in a follow-up PR)

#48 Fix unsubscription on closing/reloading application 
- Until now unsubscribe was only triggered if a diagram editor was closed but not on application close/refresh.

Fixes #48

#43 Follow-up: Update maven settings and Jenkinsbuild 
- Remove maven config and only use custom maven settings if necessary (i.e. Jenkinsbuild)
- Remove maven config and settings from project template
- Update Jenkinsbuild (do not build project-template with base framework as we have a separate build job for this)
